taui/src/components/edit-profile.js

```diff
@@ -79,6 +79,12 @@ export default class EditProfile extends PureComponent<Props> {
     }
   }
 
+  componentDidUpdate (prevProps, prevState) {
+    if (this.state.errorMessage !== prevState.errorMessage) {
```

Comment:
This will also scroll to the top on the first character change when the user goes to edit the problem field after getting the error. Changing this to `if (this.state.errorMessage) {` I think should do what we want.

Reply:
Ah, I see. That seems to work.